Sorting/quick.cpp

```diff
@@ -7,6 +7,35 @@
  *
  * Returns: null
  */
-void Sort::QuickSort(){
-    std::cout<<"[TODO @ Sorting/quick.cpp] QuickSort() needs to be implemented.\n";
+
+ //QUICK SORT ALGORITHM
```

Comment:
Unnecessary comment. Documentation will take care. Only add comment if very crucial design decision is there which others should keep in mind.

Reply:
@yatendra1999 Done.